path/common_test.ts

```diff
@@ -81,27 +78,11 @@ Deno.test({
         "./deno/std/path/mod.ts",
         "./deno/std/path/mod.ts",
       ],
-      "/",
     );
     assertEquals(actual, "./deno/std/path/mod.ts");
   },
 });
 
-Deno.test({
-  name: "windows.common() returns shared path",
-  fn() {
```

Comment:
This seemed incorrect and failed once the separator argument was removed. That's fine, as the correct corresponding test is above (name: "common() checks windows separator").